lib/tasks/posts.rake

```diff
@@ -134,3 +134,16 @@ task 'posts:remap', [:find, :replace] => [:environment] do |_,args|
   end
   puts "", "#{i} posts remapped!", ""
 end
+
+def find_each_with_recent(given_scope, recent_date: 6.months.ago)
```

Comment:
`6.months.ago` is magical to me and what works well for a not so active site might not work well for a super active site. Any thoughts on how this can be improved?